core/src/apps/benfen/sign_tx.py

```
@@ -0,0 +1,235 @@
+
+
+async def send_request_chunk(ctx: wire.Context, data_left: int) -> BenfenTxAck:
+    req = BenfenTxRequest()
+    if data_left <= 1024:
+        req.data_length = data_left
+    else:
+        req.data_length = 1024
+    return await ctx.call(req, BenfenTxAck)
+
+
+def parse_transaction(parsed_tx: dict) -> Tuple[int, str, str, int]:
```

Ruff (0.8.2)

`parse_transaction` is too complex (16 > 10)

(C901)